### PR TITLE
Feature/omb 1 testing

### PR DESCRIPTION
**Pull request by KevinWilsonOMB**
This is basically the same as the previous Pull Request which I closed, the only difference is the prefix path which I believe I have now set correctly.

### CODE REVIEW

**Comment by istankevychgsa** [comment]
Here is a demo link: https://federalist-a1f29504-db06-4d32-992e-c2dadbfe82f2.sites.pages.cloud.gov/preview/gsa/eoc/feature/OMB-1-Testing/projects/Feature_EvidencePortal_InitialCheckIn/evidence-portal/

Change your prefix path to /projects and copy your files from Feature_EvidencePortal_InitialCheckIn/evidence-portal/ directly into the /projects directory.

File: _pages/evidence-portal/2edb282b-cb5f72dd98e78832e65e.js
Comment (istankevychgsa) [inline _pages/evidence-portal/2edb282b-cb5f72dd98e78832e65e.js:1]: Please use the /projects folder instead.

**Review by istankevychgsa — CHANGES_REQUESTED** [review]
Please use /project folder

File: .gitignore [hunk 1]
```diff
@@ -7,8 +7,19 @@ node_modules
...
+assets/js/uswds-init.min.js
+assets/js/uswds.min.js
+Gemfile.lock
+package-lock.json
+assets/js/uswds-init.js
+assets/js/uswds-init.min.js
 assets/js/uswds.min.js
-assets/js/uswds.min.js.map
\ No newline at end of file
+
```
Comment (istankevychgsa) [inline .gitignore:25]: revert all changes for this file. Use .gitignore file from the **main** branch

**Review by istankevychgsa — CHANGES_REQUESTED** [review]

**Comment by KevinWilsonOMB** [comment]
Package jsons were accidently checked in, too much to resolve and will create a new PR.

**Pull request closed without merging**